R/generate_template.R

```diff
@@ -148,7 +148,7 @@ create_new_application <- function(name, location, sampleapp = FALSE, resetbutto
             }
         }
         
-        if (!(.g_sdp_installed)) {
+        if (!(.g_sdp_installed) && dashboard_plus) {
```

Comment:
no need for these extra brackets , just **!.g_sdp_installed && dashboard_plus** will be enough

Reply:
True - but I prefer more parenthesis than not, so I think this is fine.